src/components/Tabs/Tabs.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import './Tabs.css';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+
+export const Tabs = ({ tabs, tabID, onTabSelected }) => (
+  <div className="tabLinks">
+    {tabs.map(tab => (
+      <React.Fragment key={tab.id}>
+        <a
+          href="qwe"
+          className={classNames('tabLink', {
+            'tabLink--active': tabID === tab.id,
+          })}
+          onClick={event => onTabSelected(event, tab.id)}
```

Comment:
![image](https://user-images.githubusercontent.com/35385361/120029306-9659d480-bffe-11eb-8a53-87fe2aabdcf7.png)
if user clicks on the tab that already selected onTabSelected shouldn't be called